src/shared/components/create-case-filters/create-case-filters.component.ts

```diff
@@ -40,22 +40,20 @@ export class CreateCaseFiltersComponent implements OnChanges {
   filterEventControl: FormControl;
 
   constructor(
-    private orderService: OrderService
-  ) {
+    private orderService: OrderService,
+    private definitionsService: DefinitionsService,
+  ) { }
+
+  ngOnInit() {
     this.selected = {};
-    this.selectedJurisdictionCaseTypes = [];
-    this.selectedCaseTypeEvents = [];
     this.initControls();
-  }
-
-  ngOnChanges(changes?: SimpleChanges): void {
-    if (changes.jurisdictions && changes.jurisdictions.currentValue) {
-      if (this.jurisdictions.length > 0 && this.filterJurisdictionControl) {
+    this.definitionsService.getJurisdictions(CREATE_ACCESS)
```

Comment:
Can we make sure we unsubscribe in OnDestroy event cycle? Obtain the subscription to a instance variable and call unsubscribe in ngOnDestroy.